### ISSUE TITLE
[igx-grid] onSelection event comes to emit after switching back to the browser. 

### ISSUE THREAD

**Issue by tkiryu** [IgniteUI/igniteui-angular#4644]
## Description  
`onSelection` event comes to emit after switching back to the browser. 

igniteui-angular version: 7.2.4+
browser: N/A

## Steps to reproduce  

1. Go to https://igx-grid-onselection-issue-726.stackblitz.io
2. Open console.
3. Press Alt+Tab to switch to another window(Or click another window).
4. Press Alt+Tab to switch back to the browser(Or click the browser).

## Result  
After step 4, `onSelection` event emits.

## Expected result  
After step 4, `onSelection` event should not emit.

Until v7.2.3, `onSelection` event won't emit. Here is a sample.
https://stackblitz.com/edit/igx-grid-onselection

**Comment by rkaraivanov** [comment]
Duplicate of #4625

**Comment by tkiryu** [comment]
@rkaraivanov , I'm sorry about that. Thanks!
